### PR TITLE
Fixes #985 - Fix floating point math for 'receive' calculations

### LINKED ISSUE #985: [QA] Many extra decimals are shown in amount in To section
## Describe the current behavior in detail
[QA] Many extra decimals are shown in amount in To section.

Checked from https://cctp-only--wormhole-connect.netlify.app/ [Mainnet] with macOS 12.6.7 and Google Chrome Version 116.0.5845.179 (Official Build) (x86_64).

Steps:
1. Select Ethereum in From section
2. Select Arbitrum in To section
3. Connect Metamask wallets
4. Enter amount 1.12
5. Observe Amount in To section (actual result: 0.1200000000000001)

https://github.com/wormhole-foundation/wormhole-connect/assets/10746654/3c0a6fb2-b4e4-4bd9-b836-a3f8e2b7c705

## What is the expected/desired behavior
Correct Amount is displayed in To section.

### PR DESCRIPTION
**Pull request by derpy-duck**
Fixes #985

### CODE REVIEW

File: wormhole-connect/src/utils/routes/relay.ts [hunk 1]
```diff
@@ -176,10 +176,16 @@ export class RelayRoute extends BridgeRoute {
     routeOptions: RelayOptions,
   ): Promise<number> {
     if (!sendAmount) return 0;
+
+    const { toNativeToken, relayerFee } = routeOptions;
+
+    // floating point math
+    const DECIMALS = 10 ** 8;
```
Comment (evan-gray) [inline wormhole-connect/src/utils/routes/relay.ts:183]: Is this always 8 decimals? Seems dangerous to hardcode.

**Review by evan-gray — CHANGES_REQUESTED** [review]
This seems like a good time to add some unit tests for these calculations. I’m also concerned by hardcoding the decimals, though maybe could be convinced if there’s a very good reason it’s okay.

File: wormhole-connect/src/utils/routes/relay.ts (hunk 1 above)
Comment (derpy-duck) [reply wormhole-connect/src/utils/routes/relay.ts:183]: This is the token bridge route - which only supports up to 8 decimals right (this serves as an upper bound on # of decimals)

Comment (evan-gray) [reply wormhole-connect/src/utils/routes/relay.ts:183]: ah, I see the division at the bottom (hence why you mentioned "up to").

**Review by evan-gray — APPROVED** [review]
As discussed with @derpy-duck, I am firmly against using `number` for these sorts of calculations, but I agree that change will require a larger re-write.

**Pull request merged**